account_invoice_inter_company/views/account_invoice_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="invoice_supplier_form" model="ir.ui.view">
+        <field name="name">account.invoice.supplier.form - HACK: Extra fields for odoo.tests.Form() to work</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']/tree" position="inside">
+                <field name="company_id" invisible="1" readonly="0"/>
```

Comment:
Already in the main view. Could it be causing the error in travis?
https://github.com/odoo/odoo/blob/12.0/addons/account/views/account_invoice_view.xml#L492